extensions/import/src/wizard/flatFileWizard.ts

```diff
@@ -39,7 +39,12 @@ export class FlatFileWizard {
 		let pages: Map<number, ImportPage> = new Map<number, ImportPage>();
 
 		let connectionId = (await azdata.connection.getCurrentConnection())?.connectionId ??
-			(await azdata.connection.openConnectionDialog())?.connectionId;
+			(await azdata.connection.openConnectionDialog(['MSSQL']))?.connectionId;
+
+		if ((await azdata.connection.getCurrentConnection()).providerId !== 'MSSQL') {
```

Comment:
If we already have a connection there's no reason to call this twice - just save the connection from the above line and use that instead.